workspace/controller.py

```diff
@@ -21,7 +22,10 @@ def create():
 @workspaces_bp.route('/', methods=['GET'])
 @workspace_bp.route('/', methods=['GET'])
 def get(workspace_id: str = None):
-    workspaces = ws_service.get(workspace_id)
+    user_token = request.headers.get('Authorization').split()[1]
+    user_token = base64.b64decode(user_token)
+
+    workspaces = ws_service.get(workspace_id, user_token)
```

Comment:
functionally this doesn't matter at all but what if we got the user here rather than passing the token down to the service and data layers? (it just feels a little odd to me for the data/service layers to have to worry about tokens)